libimage/import.go

```diff
@@ -23,6 +23,9 @@ type ImportOptions struct {
 	CommitMessage string
 	// Tag the imported image with this value.
 	Tag string
+	// Pref Platform for the image
+	OS   string
+	Arch string
```

Comment:
Please add comments to both fields.  This way the comments will show up in doc generators and IDEs.

`// Overwrite OS/Arch of imported image.

Reply:
will do, thanks